plugins/Marketplace/Marketplace.php

```diff
@@ -227,6 +229,32 @@ public function removePluginTrialRequest(string $pluginName): void
         }
     }
 
+    public function isInstallAllPaidPluginsVisible(): bool
+    {
+        $consumer = StaticContainer::get('Piwik\Plugins\Marketplace\Consumer');
```

Comment:
To test add below code
```suggestion
       return true;
        $consumer = StaticContainer::get('Piwik\Plugins\Marketplace\Consumer');
```